core/src/main/java/org/elasticsearch/cluster/DiskUsage.java

```diff
@@ -33,7 +33,7 @@
 /**
  * Encapsulation class used to represent the amount of disk used on a node.
  */
-public class DiskUsage implements ToXContent, Writeable {
+public class DiskUsage implements ToXContentFragment, Writeable {
```

Comment:
can you make toShortXContent package private here and maybe remove the Params argument that's unused?